backend/src/main/kotlin/rocks/didit/sefilm/domain/Participant.kt

```diff
@@ -0,0 +1,7 @@
+package rocks.didit.sefilm.domain
+
+open class Participant(val userID: UserID)
```

Comment:
Yes I tried sealed first, altought I ran into som serialization-trouble when extracting data from Mongo. Not sure how to resolve it.